doc/conf.py

```diff
@@ -62,7 +62,11 @@
 copybutton_prompt_text = r">>> |\.\.\. |\$ |In \[\d*\]: | {2,5}\.\.\.: | {5,8}: "
 copybutton_prompt_is_regexp = True
 
-intersphinx_mapping = {"https://pennylane.ai/qml/": None}
+intersphinx_mapping = {
+    "demo": ("https://pennylane.ai/qml/", None),
+    "catalyst": ("https://docs.pennylane.ai/projects/catalyst/en/stable", None)
+}
+
```

Comment:
This will allow us to cross reference the catalyst docs from the PL docs, without hardcoding in absolute URLs

Reply:
:star_struck: